### PR TITLE
Bugfix/relative url route

### PR DESCRIPTION
**Pull request by danielmurray**

### CODE REVIEW

File: pyramid_orb/api.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def process(self, request):
             output = {}
             for schema in schemas:
                 dbname = schema['dbname']
-                schema['urlRoot'] = request.host_url + request.path.rstrip('/') + '/' + dbname
+                schema['urlRoot'] = request.path.rstrip('/') + '/' + dbname
```
Comment (coolnay309) [inline pyramid_orb/api.py]: Nice!

Comment (erichulser) [inline pyramid_orb/api.py]: Why are we changing this?  This was specifically to support CORS.  We should come up with a way to support both (look at the requesting path and determine if it needs to be the full url or not)

Comment (danielmurray) [inline pyramid_orb/api.py]: @erichulser, well the issue was that when we where serving it from dev, the url route was prepending a https uri scheme name

Comment (erichulser) [inline pyramid_orb/api.py]: that's interesting that host_url was prepending https....weird....

**Pull request merged**